services/cd-service/pkg/repository/repository.go

```diff
@@ -471,12 +473,23 @@ func (r *repository) Apply(ctx context.Context, transformers ...Transformer) err
 		r.writesDone = r.writesDone + uint(len(transformers))
 		r.maybeGc(ctx)
 	}()
-	eCh := r.applyDeferred(ctx, transformers...)
-	select {
-	case err := <-eCh:
-		return err
-	case <-ctx.Done():
-		return ctx.Err()
+	const deferIt = true
+	if deferIt {
```

Comment:
Is this intended to be here?

Reply:
no!

Reply:
I removed it.
(that was an attempt to compare the benchmark between deferred/non-deferred operations. It was not finished)